src/components/widgets/accounts-cash-balance/accounts-cash-balance.directive.coffee

```diff
@@ -0,0 +1,132 @@
+#
+# Component generated by Impac! Widget Generator!
+# TODO:
+# - colour shades by account sub-type (don't spend too much time on this one / not that important)
+# - check if a different chart can be used for better time period selection
+#
+module = angular.module('impac.components.widgets.accounts-cash-balance', [])
+module.controller('WidgetAccountsCashBalanceCtrl', ($scope, $q, $timeout, $filter, ImpacTheming) ->
+
+  w = $scope.widget
+
+  # Define settings
+  # --------------------------------------
+  $scope.orgDeferred = $q.defer()
+  $scope.timePeriodDeferred = $q.defer()
+
+  settingsPromises = [
+    $scope.orgDeferred.promise,
+    $scope.timePeriodDeferred.promise
+  ]
+
+  # Widget specific methods
+  # --------------------------------------
+  w.initContext = ->
+    if $scope.isDataFound = w.content?
```

Comment:
For bolt widgets, `content` will always be defined - should we check if content.chart is defined instead?
TODO: when data not found, query the API again to load stub data.